protocol-rpc/build.rs

```diff
@@ -2,19 +2,25 @@
 //  SPDX-License-Identifier: Apache-2.0
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    tonic_build::compile_protos("proto/common.proto")
-        .unwrap_or_else(|e| panic!("Failed to compile protos {:?}", e));
-    tonic_build::compile_protos("proto/privateid.proto")
-        .unwrap_or_else(|e| panic!("Failed to compile protos {:?}", e));
-    tonic_build::compile_protos("proto/privateidmultikey.proto")
-        .unwrap_or_else(|e| panic!("Failed to compile protos {:?}", e));
-    tonic_build::compile_protos("proto/crosspsi.proto")
-        .unwrap_or_else(|e| panic!("Failed to compile protos {:?}", e));
-    tonic_build::compile_protos("proto/crosspsixor.proto")
-        .unwrap_or_else(|e| panic!("Failed to compile protos {:?}", e));
-    tonic_build::compile_protos("proto/pjc.proto")
-        .unwrap_or_else(|e| panic!("Failed to compile protos {:?}", e));
-    tonic_build::compile_protos("proto/suidcreate.proto")
-        .unwrap_or_else(|e| panic!("Failed to compile protos {:?}", e));
+    let grpc_path = "proto";
+    let proto_files = &[
+        "common.proto",
+        "privateid.proto",
+        "privateidmultikey.proto",
+        "crosspsi.proto",
+        "crosspsixor.proto",
+        "pjc.proto",
+        "suidcreate.proto",
+    ];
+
+    tonic_build::configure()
```

Comment:
Can you make `format(true)` or just not call `format` - its `true` by default - it generates human readable stubs that are super helpful for developers